### PR TITLE
Update associated projects

### PR DESCRIPTION
**Pull request by hqhq**
Now we have ocitools in opencontainers, thanks @mrunalp .

Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

File: implementations.md [hunk 1]
```diff
@@ -11,11 +11,8 @@ If you know of any associated projects that are not listed here, please file a p
...
 
-## Bundle authoring
+## Testing & Tools
 
 * [kunalkushwaha/octool](https://github.com/kunalkushwaha/octool) - A config linter and validator.
-* [mrunalp/ocitools](https://github.com/mrunalp/ocitools) - A config generator.
-
-## Testing
-
+* [opencontainers/ocitools](https://github.com/opencontainers/ocitools) - A config generator and runtime testing framework.
```
Comment (wking) [inline implementations.md]: I'm not clear on intended scoping for this project, but there's an in-flight PR for bundle validation as well (opencontainers/ocitools#20) that hasn't been closed as “out of scope”.  So probably list bundle testing as well, unless we want to file a follow-up PR adding it if/when the ocitools PR lands.

Comment (hqhq) [inline implementations.md]: I think the plan is we(huawei guys) will backport as many features as we can from [oct](https://github.com/huawei-openlab/oct) to [ocitools](https://github.com/opencontainers/ocitools), then we can remove oct here and rewrite the scope of ocitools. Or maybe @mrunalp can give a better description of scope for ocitools now?

Comment (liangchenye) [inline implementations.md]: Agree, 'testing&tools' is a better description here.  

I think @mrunalp agreed ocitools should cover three areas currently.(I cannot find the related conversation in issue/PR)
1. generate
2. bundle validate
3. runtime validate

**Comment by mrunalp** [comment]
@liangchenye Yes, those are the initial tools and we could always add more tooling as the project evolves.

**Comment by mrunalp** [comment]
@hqhq Can you update the PR?

**Comment by wking** [comment]
While we're touching these references, maybe drop in a reference to
jfrazelle/riddler (mentioned on IRC on Monday [1]).

[1]: http://ircbot.wl.linuxfoundation.org/eavesdrop/%23opencontainers/%23opencontainers.2016-03-14.log.html#t2016-03-14T19:28:18

**Comment by crosbymichael** [comment]
Where do we draw the line of what is added to this list?  I don't see it as a scale-able thing to have in the repo.  Are we just going to link to anything and everything?

**Comment by mrunalp** [comment]
@crosbymichael Maybe we could just provide a pointer to opencontainer/ocitools (Since it will be supported and updated to keep up with the spec) and keep the rest in a wiki?

**Comment by crosbymichael** [comment]
Ya, it would probably be better to just have the official repositories here if anything at all.  If you think about it, if we add just any repo what uses or consumes the spec it will be a never ending battle of one line prs to add some repo that we know nothing about.  The title is "associated" projects implying that we are associated with these things when that is not the case.

I wiki could scale better but I have not used github wikis so I don't know how well it works.

**Comment by wking** [comment]
On Wed, Mar 16, 2016 at 03:01:07PM -0700, Michael Crosby wrote:
> If you think about it, if we add just any repo what uses or consumes
> the spec it will be a never ending battle of one line prs to add
> some repo that we know nothing about.

I'm fine with an open door (punting on curation) until we have at
least twenty or so referenced projects here ;).  Any consumer that a
spec maintainer links to on IRC sounds like fair game for curation
anyway.

**Comment by hqhq** [comment]
Updated the description of ocitools.

**Comment by vbatts** [comment]
sadly, the CI failures are unrelated. I opened #350 to track that separately.

**Comment by vbatts** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**